apps/angular/crud/src/app/ui/todo/todo.component.html

```diff
@@ -0,0 +1,20 @@
+
+<div class="flex items-center justify-between gap-5 p-2 hover:bg-slate-200">
+  <div class="w-30vw align-middle">
+    {{todo.title}}
+  </div>
+  <div class="flex gap-3">
+    <button class="bg-blue-500 hover:bg-blue-700 text-white  py-2 px-4 rounded-full hover:shadow-lg
+      {{loaderService.isLoading() ? 'opacity-50 cursor-not-allowed' : ''}}"
+      (click)="updateTodo(todo)"
+      [disabled]="loaderService.isLoading()">
+      Update
+    </button>
+    <button class="bg-pink-500 hover:bg-pink-700 text-white  py-2 px-4 rounded-full hover:shadow-lg
+      {{loaderService.isLoading() ? 'opacity-50 cursor-not-allowed' : ''}}"
```

Comment:
I'd rather refactor that part `{{loaderService.isLoading() ? 'opacity-50 cursor-not-allowed' : ''}}` 
with something like this `[ngClass]="{ 'opacity-50 cursor-not-allowed': loaderService.isLoading() }"`

Reply:
Thanks for attention and advice. I did not think of this option. Nice solution instead of using string interpolation.

Reply:
ngClass will get deprecated and is not very optimized, 
you should use `[class.opacity-50]="loaderService.isLoading()"` and so on.

Reply:
or add `[class.is-loading]="loaderService.isLoading()"` and in styles `.is-loading { @apply opacity-50 cursor-not-allowed; }`